gemini-resin-archetype/src/main/resources/archetype-resources/src/main/webapp/WEB-INF/configuration/hikari.properties

```diff
@@ -0,0 +1,9 @@
+# This is only used if Application.constructConnectorFactory() returns a
+# HikariCPConnectorFactory. Otherwise see the Gemini configuration files
+# for database connection settings.
+dataSourceClassName=org.postgresql.ds.PGSimpleDataSource
+dataSource.user=${databaseUsername}
+dataSource.password=${databasePassword}
+dataSource.databaseName=${databaseName}
+dataSource.portNumber=${databasePort}
+dataSource.serverName=${databaseHost}
```

Comment:
This is one of those things that ends up confusing down the line. This states that if I configure my application, which may have database credentials in several gemini-specific configuration files, to use HikariCP then the expectation is that this `hiarki.properties` file will be used while the gemini-specific ones will be ignored as it pertains to database configuration.

So, one could easily find himself in the situation where a project switches over and the production (as an example) config says one thing but the hikari says another, and a new dev would be very confused.

I don't have a perfect solution to this, but it seems likely to be confusing unless we do [something] to make it less so.

Reply:
Good point. Recommend discarding this unless we make gemini-hikaricp the default in our other conversation.